distributed/distributed.yaml

```diff
@@ -24,6 +24,9 @@ distributed:
     pickle: True            # Is the scheduler allowed to deserialize arbitrary bytestrings
     preload: []
     preload-argv: []
+    # Assume this duration ("15m", "2h") for all tasks with unknown durations
+    # This can be useful to change occupancy estimations in Adaptive scenarios
+    task-duration: null
```

Comment:
Should this have a non-null value?  What do we do currently?

So far we've been limiting comments for configuration values to be a bit shorter and be on the same line.  If we choose to change convention here and have much larger comments then we should probably do that across the file.  For now I recommend that we try to make the comment more brief if we can.